server/src/domain/asset/asset.service.spec.ts

```diff
@@ -132,6 +179,14 @@ describe(AssetService.name, () => {
             );
           });
         }
+
+        it('should be sorted (valid)', () => {
+          expect(valid).toEqual([...valid].sort());
+        });
+
+        it('should be sorted (invalid)', () => {
+          expect(valid).toEqual([...valid].sort());
```

Comment:
These should be invalid actually

Reply:
Thanks, I think I copied this from #3317 and missed it.